bedrock/base/templates/macros.html

```diff
@@ -423,20 +423,35 @@ <h3>{{ _('Your download link was sent.') }}</h3>
     <input type="hidden" name="utm_{{ attr }}" value="{{ val }}" id="fxa-email-form-utm-{{ attr }}" />
   {% endfor %}
 
-    <p>
+    <p class="fxa-email-form-intro">
+    {% if intro_text %}
+      {{ intro_text }}
+    {% else %}
       {{ _('<strong>Enter your email</strong> to access Firefox Accounts.') }}
+    {% endif %}
     </p>
 
-    <label for="fxa-email-field">{{ _('Email address') }}</label>
-    <input type="email" name="email" id="fxa-email-field" class="fxa-email-field" placeholder="user@example.com" required />
-
-    <p class="disclaimer">
+    <p class="agreement">
     {% trans url1='https://accounts.firefox.com/legal/terms', url2='https://accounts.firefox.com/legal/privacy' %}
       By proceeding, you agree to the <a href="{{ url1 }}">Terms of Service</a> and
       <a href="{{ url2 }}">Privacy Notice</a>.
     {% endtrans %}
     </p>
 
-    <button type="submit" class="button button-blue" id="fxa-email-form-submit">{{ _('Continue') }}</button>
+    <div class="fxa-email-field-container">
+      <p class="field">
+        <label for="fxa-email-field">{{ _('Email address') }}</label>
+        <input type="email" name="email" id="fxa-email-field" class="fxa-email-field" placeholder="user@example.com" required>
+      </p>
+
+      <button type="submit" class="button {{ button_class }}" id="fxa-email-form-submit">
```

Comment:
Adding an option to pass `button_class` here is super helpful 👍. Could we use it to replace the existing `.button` class instead though perhaps? Just thinking in terms of protocol pages that may use this: we could want to add `.mzp-c-button` but not have the existing `.button` styling?

Reply:
✅